platforms/riscv32/features/BUILD.bazel

```diff
@@ -46,6 +46,7 @@ feature(
                     flags = [
                         "-Os",
                         "-g",
+                        "-gdwarf-4",
```

Comment:
This is needed for compatibility with the GNU bfd.ld linker version we're still using. We can remove it and start using the new default of DWARF 5 once we adopt LLD.